component.yaml

```
@@ -0,0 +1,70 @@
+platform: Common
+buildPlatform: ReactNative
+displayName: Appcircle CodePush
+description: "Release a React Native update to Appcircle CodePush."
+inputs:
+- key: "AC_REPOSITORY_DIR"
+  defaultValue: "$AC_REPOSITORY_DIR"
+  isRequired: true
+  title: Repository Directory
+  description: "Relative path of the React Native project."
+  helpText:
+- key: "AC_CODE_PUSH_TOKEN"
+  defaultValue: "$AC_CODE_PUSH_TOKEN"
+  isRequired: true
+  title: Appcircle Personal Access Token
+  description: "Appcircle personal access token. You can create a new token in the Appcircle dashboard under `Settings > Security > Personal Access Token`."
+  helpText:
+- key: "AC_CODE_PUSH_SERVER_URL"
+  defaultValue: "$AC_CODE_PUSH_SERVER_URL"
+  isRequired: false
+  title: Appcircle CodePush Server URL
+  description: "Appcircle CodePush server URL. Leave it empty if you do not use self-hosted server."
+  helpText:
+- key: "AC_CODE_PUSH_AUTH_URL"
+  defaultValue: "$AC_CODE_PUSH_AUTH_URL"
+  isRequired: false
+  title: Appcircle CodePush Auth URL
+  description: "Appcircle CodePush auth URL. Leave it empty if you do not use self-hosted server."
```

```suggestion
  description: "Appcircle CodePush auth URL. Leave it empty if you do not use self-hosted server. Ignore this if you are not a self-hosted Appcircle user."
```

1. IMO the word "Empty" may make the user feel that they have to empty this place, but "Ignor" is more appropriate since they do not need to do anything.
2. It might be more explanatory to state that self-hosted is Appcircle self-hosted.